src/dataflow/src/sink/avro_ocf.rs

```diff
@@ -36,7 +36,7 @@ pub fn avro_ocf<G>(
         v
     });
     let (schema, columns) = {
-        let encoder = Encoder::new(None, desc, false);
+        let encoder = AvroEncoder::new(None, desc, false, None, None);
```

Comment:
This made me remember that the old `Encoder` is weird and has this double-use we see here. Where the schema IDs are `None`, which feels iffy.

Reply:
Agreed! This bothered me too... I think I'll separate them out!